src/main/java/anab/Person/Trainer.java

```diff
@@ -0,0 +1,105 @@
+package anab.Person;
+
+import java.nio.MappedByteBuffer;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+
+public class Trainer extends Person {
+    private String specialization;
+    private int experienceYears;
+    private Map<String, Integer> feedbackMarks = new HashMap<>();
+
+    public Trainer(String firstName, String lastName, String gender, int age, String emailAddress, boolean isEmailRestricted, String specialization, int experienceYears, Map<String, Integer> feedbackMarks) {
+        super(firstName, lastName, gender, age, emailAddress, isEmailRestricted);
+        this.specialization = specialization;
+        this.experienceYears = experienceYears;
+        this.feedbackMarks = feedbackMarks;
+    }
+
+    // see the specialization of a trainer
+    public String getSpecialization() {
+        return this.specialization;
+    }
+
+    //add specialization of a trainer
+    public void setSpecialization(String specialization) {
+        this.specialization = specialization;
+    }
+
+    //see the years of experience of a trainer
+    public int getExperience() {
+        return this.experienceYears;
+    }
+
+    //Add years of experience for a trainer
+    public void addExperience(String spec, int expYears) {
```

Comment:
as we discussed, this unused method will be removed

Reply:
accepted :-|